src/configuration/SSH/OpenSSH/7.3/sshd_config

```diff
@@ -0,0 +1,118 @@
+#-----------------------------
+# Daemon
+#-----------------------------
+
+# Specifies the port number that sshd(8) listens on.
+# The default is 22.
+#Port 1022
+
+# Specifies the local addresses sshd(8) should listen on.
+# The default is to listen on all local addresses.
+#ListenAddress 192.168.23.42
+
+# Specifies a file containing a private host key used by SSH.
+# The default is
+## /etc/ssh/ssh_host_dsa_key,
+## /etc/ssh/ssh_host_ecdsa_key,
+## /etc/ssh/ssh_host_ed25519_key and
+## /etc/ssh/ssh_host_rsa_key for protocol version 2.
+HostKey /etc/ssh/ssh_host_ed25519_key
+HostKey /etc/ssh/ssh_host_rsa_key
+
+# Specifies the maximum number of concurrent unauthenticated connections
+# to the SSH daemon.
+# The default is 10:30:100.
+MaxStartups 3
+
+# Specifies the maximum number of authentication attempts permitted per
+# connection.
+# The default is 6.
+MaxAuthTries 1
+
+# Specifies the maximum number of open shell, login or subsystem (e.g. sftp)
+# sessions permitted per network connection.
+# The default is 10.
+MaxSessions 6
+
+#-----------------------------
+# Authentication
+#-----------------------------
+
+# The server disconnects after this time if the user has not
+# successfully logged in.
+# The default is 120 seconds.
+LoginGraceTime 60
+
+# Specifies whether root can log in using ssh(1).
+# The default is "prohibit-password".
+PermitRootLogin no
+
+# Specifies the file that contains the public keys that can be used for
+# user authentication.
+# The default is ".ssh/authorized_keys .ssh/authorized_keys2".
+AuthorizedKeysFile .ssh/authorized_keys
+
+# Specifies whether password authentication is allowed.
+# The default is "yes".
+PasswordAuthentication no
```

Comment:
The name of this project is **better** crypto, not **good** crypto.

`PasswordAuthentication` should be discouraged:

> It is also recommended to disable password logins entirely. This will greatly increase security, see  [#Force public key authentication](https://wiki.archlinux.org/index.php/Ssh#Force_public_key_authentication) for more information.